src/DataViews/SamplingReportDataViewTest.cpp

```diff
@@ -523,6 +600,40 @@ TEST_F(SamplingReportDataViewTest, ContextMenuActionsAreInvoked) {
     CheckExportToCsvIsInvoked(context_menu, app_, view_, expected_contents);
   }
 
+  // Export Callstack Events to CSV
+  {
+    EXPECT_CALL(sampling_report_, GetCallstackData).WillRepeatedly(ReturnRef(*kCallstackData));
+    EXPECT_CALL(sampling_report_, GetSelectedCallstackIds)
+        .Times(4)
+        .WillOnce(Return(std::nullopt))
+        .WillOnce(Return(std::nullopt))
+        .WillOnce(Return(kSelectedCallstackIds))
+        .WillOnce(Return(kSelectedCallstackIds));
+
+    // all threads, no active selection
```

Comment:
Nit: Capitalize the initial letter.

If you use the lower case to indicate this is a sub test case for "Export Callstack Events to CSV", then maybe use the complete sentence "Test the case of all threads, no active selection."